internal/distro/rhel85/distro.go

```diff
@@ -213,7 +213,10 @@ func (t *imageType) PackageSets(bp blueprint.Blueprint) map[string]rpmmd.Package
 	if timezone != nil {
 		bpPackages = append(bpPackages, "chrony")
 	}
-	mergedSets["packages"] = mergedSets["packages"].Append(rpmmd.PackageSet{Include: bpPackages})
+
+	// repsolve bp packages separately
```

Comment:
resolve

Reply:
Fixed (actually, wanted to say 'depsolve').  Thanks!